pkgs/development/python-modules/suds/default.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  isPyPy,
+  pytestCheckHook,
+  python311Packages,
+}:
+
+buildPythonPackage rec {
+  pname = "suds";
+  version = "1.1.2";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```

`format` has been removed from the [docs](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters), which now refer to `pyproject = true;`.

Also, add `build-system = [ setuptools ];` if necessary.